docs/config/README.md

```diff
@@ -3265,22 +3265,23 @@ To enable it, set `disabled` to `false` in your configuration file.
+| Option                      | Default                                                                       | Description                                                           |
+| --------------------------- | ----------------------------------------------------------------------------- | --------------------------------------------------------------------- |
+| `format`                    | `"[$symbol$status]($style) "`                                                 | The format of the module                                              |
+| `symbol`                    | `"✖"`                                                                         | The symbol displayed on program error                                 |
+| `success_symbol`            | `""`                                                                          | The symbol displayed on program success                               |
+| `not_executable_symbol`     | `"🚫"`                                                                         | The symbol displayed when file isn't executable                       |
+| `not_found_symbol`          | `"🔍"`                                                                         | The symbol displayed when the command can't be found                  |
+| `sigint_symbol`             | `"🧱"`                                                                         | The symbol displayed on SIGINT (Ctrl + c)                             |
+| `signal_symbol`             | `"⚡"`                                                                         | The symbol displayed on any signal                                    |
+| `style`                     | `"bold red"`                                                                  | The style for the module.                                             |
+| `recognize_signal_code`     | `true`                                                                        | Enable signal mapping from exit code                                  |
+| `map_symbol`                | `false`                                                                       | Enable symbols mapping from exit code                                 |
+| `pipestatus`                | `false`                                                                       | Enable pipestatus reporting                                           |
+| `pipestatus_separator`      | <code>&vert;</code>                                                           | The symbol used to separate pipestatus segments                       |
+| `pipestatus_format`         | `\\[$pipestatus\\] => [$symbol$common_meaning$signal_name$maybe_int]($style)` | The format of the module when the command is a pipeline               |
+| `pipestatus_segment_format` | `None`                                                                        | When specified, replaces `format` when formatting pipestatus segments |
```

Comment:
We leave `None` defaults empty, because TOML doesn't have any equivalent.
```suggestion
| `pipestatus_segment_format` |                                                                        | When specified, replaces `format` when formatting pipestatus segments |
```